README.md

```diff
@@ -37,7 +37,7 @@ var coords = formatcoords(27.725499,-18.024301);
 ```
 
 ```
-coords.format([format, lat/lon separator])
+coords.format([format, lat/lon separator, number of decimal places])
```

Comment:
I think at this point we should switch to a different method signature:
```
coords.format([format], [params])
```
Where format and params are optional, and params is an object with optional parameters: 
```
{
   latLonSeparator (string, defaults to ' ')
   decimalPlaces (int, defaults to 5)
}
```
Following cases should be handled (also in tests)
- defaults values when `params` is not specified and when keys inside `params` are not specified
- case when `format` is omitted (type detection)
- case when second parameter is a string, in which case assume that it's lat/lon spearator (for retro compatibility)